Meta/lb-cherry-picks.py

```diff
@@ -0,0 +1,514 @@
+serenity_cherry_picks.add('e64ac8c17732bdb97689644b43d07f61d3f6cab2')
+serenity_cherry_picks.add('3d7c8246072997038ff8ad41875cd3b4604bcfd2')
+serenity_cherry_picks.add('a7b1a9ded759fdf27cace181128c9cd0a5528d44')
+serenity_cherry_picks.add('67749300c301faec825e6f0b215f4eae89adc3b6')
+serenity_cherry_picks.add('94c2b85959a7afa0168f1e1184bedb87827666ee')
+serenity_cherry_picks.add('d20f1a99f87d0477ada1a112a1d55d6ed47eadf9')
+serenity_cherry_picks.add('40fcb00c142457615a840a6b5c78d21b6ac7159b')
+serenity_cherry_picks.add('cf7937e369c85472fef72bcacbc11c1ad172e206')
+serenity_cherry_picks.add('b92bd12a8e3b372ba374bfbebd91972b3d5d7a00')
+serenity_cherry_picks.add('991759f453349baf754ec3aa22934fae66377ab2')
+serenity_cherry_picks.add('8217a7772878d407452858ecc915202941f3e01f')
+serenity_cherry_picks.add('6e419db26c3750609fb4eabcb8157495ad8d3d16')
+
+# https://github.com/SerenityOS/serenity/pull/24744 cherry-picked
+# https://github.com/LadybirdBrowser/ladybird/pull/558
+# https://github.com/LadybirdBrowser/ladybird/pull/559
+# https://github.com/LadybirdBrowser/ladybird/pull/563
+# https://github.com/LadybirdBrowser/ladybird/pull/582
+# https://github.com/LadybirdBrowser/ladybird/pull/591
+# https://github.com/LadybirdBrowser/ladybird/pull/603
+# https://github.com/LadybirdBrowser/ladybird/pull/605
+# https://github.com/LadybirdBrowser/ladybird/pull/594
+# https://github.com/LadybirdBrowser/ladybird/pull/614
+# https://github.com/LadybirdBrowser/ladybird/pull/615
+# but somehow got the commit hashes wrong.
+# (It also cherry-picked #635 and #646, but got those hashes right.)
+serenity_cherry_picks.add('625fbc8085debf5f145ec1e9de24854fc9573834')
+serenity_cherry_picks.add('d07cf26894bcaa5791ed3b7715b94e25227b2bda')
+serenity_cherry_picks.add('420a626554ac595d6be68a590b8ad32b42db8ef8')
+serenity_cherry_picks.add('da8633b2d0ab3b9d8f1cdad39a8ad85ca2accf03')
+serenity_cherry_picks.add('524e09dda180a67e9504d12668e20ada0c278f04')
+serenity_cherry_picks.add('3efb11f5d8316ee81924b123f008a21c52afcc83')
+serenity_cherry_picks.add('13a8c2a79df59502da0d3f40e4844b10f3a8f041')
+serenity_cherry_picks.add('4c7ef01b4479066016118306a67949480bb1ac44')
+serenity_cherry_picks.add('3a0f80bbae4c291df1e1847b5ce2058221482d81')
+serenity_cherry_picks.add('f0a306fe500c392a5207b3bdd88f781230c2cbfe')
+serenity_cherry_picks.add('906fa0482236c050e8386f1e1969edf531e5e257')
+
+# Define a list of pull request IDs that should never be merged
+# and the reasons why they shouldn't be merged.
+# We fairly likely don't want the PRs in here (but it isn't 100% set
+# in stone for some of them). But if a PR is not in here, it doesn't
+# necessarily mean that we want it: It might just not be triaged yet.
+# If we want some commits of a PR but not all, the PR should not be
+# in this list -- but some commits of the PR we don't want in
+# never_merge_commits further down instead.
+never_merge_prs = {
+    "https://github.com/LadybirdBrowser/ladybird/pull/16": "Ladybird-specific",
```

Comment:
What about PRs 1-15?

Reply:
Good question! PR15 is actually here, two lines down. I sorted these by merge time, not by PR number, at the start. That's a pain to maintain though, so I moved it to sorted now.

What about PRs 1-14? It looks like the merged commits there went to a different branch for some reason, and they show up as `Commit: https://github.com/LadybirdBrowser/ancient-history/commit/dd1fbd3513` and don't have a `Pull-request:` footer. I annotated commits in `never_merge_commits` with their PR number.

Reply:
Oh maybe at some point they were trying to revive that repo? Well as long as those don't show up then it's ok